File: src/main/java/de/lukaseichler/recurrentneuralnetwork/activation/LogActivation.java
package de.lukaseichler.recurrentneuralnetwork.activation;

/**
 * @author leichler
 */
public class LogActivation implements ActivationFunction {
    @Override
    public double apply(double input) {
        return 1 / (1 + Math.pow(Math.E, -input));
    }
}

File: src/main/scala/de/lukaseichler/recurrentneuralnetwork/activation/LogActivation.scala
package de.lukaseichler.recurrentneuralnetwork.activation

/**
 * @author Lukas Eichler

 */
class LogActivation extends ActivationFunction {
  override def apply(input: Double): Double = {
    1 / (1 + Math.pow(Math.E, -input))
  }
}
